src/task_manager.rs

```diff
@@ -20,6 +24,7 @@ pub(crate) struct TaskManager {
     exchange_rates: Arc<Mutex<Vec<ExchangeRate>>>,
     data_store: Arc<Mutex<DataStore>>,
     sdk: Arc<BreezServices>,
+    lsp_fee: Arc<Mutex<Option<LspFee>>>,
```

Comment:
Instead of storing an `LspFee`, I would store the `OpeningFeeParams` provided by Breez. That way, we avoid having to use the following expressions, which are a bit weird :) 
`cheapest_opening_fee.min_msat.to_amount_up(&None)`
and 
`(lsp_fee_cache.channel_minimum_fee.sats * 1000)
                .to_amount_up(exchange_rate)`
This way, you just have to convert from `msat` to `Amount` in `get_lsp_fee()`

```suggestion
    lsp_fee: Arc<Mutex<Option<OpeningFeeParams>>>,
```